src/accordion.tsx

```diff
@@ -0,0 +1,29 @@
+import { ComponentChildren, VNode } from "preact";
+
+interface AccordionProps {
+  id: string;
+  title: string | VNode;
+  children?: ComponentChildren;
+}
+
+function Accordion({ id, title, children }: AccordionProps): VNode {
```

Comment:
Since the `id` isn't used externally, we could generate it internal to the component?

We use this in the IdP, for example: https://github.com/18F/identity-idp/blob/main/app/javascript/packages/document-capture/hooks/use-instance-id.js

Reply:
I like it! Will copy

Reply:
b5ab799edb8dbf80171d4e08e11e56196f452269, 1af1cb5ad44b3f9bdf88e2758f1c5054be9eea42